Archive/app.js

```diff
@@ -0,0 +1,137 @@
+console.log("Sanity Check: JS is working!");
+var $venuesList;
+var allVenues = [];
+
+$(document).ready(function(){
+
+  $venuesList = $('#venueTarget');
+  $.ajax({
+    method: 'GET',
+    url: '/api/venues',
+    success: handleSuccess,
+    // error: handleError
+  });
+
+  // $('#venue-form').on('submit', function(e) {
+  //   e.preventDefault();
+  //   $.ajax({
+  //     method: 'POST',
+  //     url: '/api/venues',
+  //     data: $(this).serialize(),
+  //     success: newVenueSuccess,
+  //     // error: newBookError
+  //   });
+  // });
+  //
+  // $venuesList.on('click', '.deleteBtn', function() {
+  //   console.log('clicked delete button to', '/api/venues/'+$(this).attr('data-id'));
+  //   $.ajax({
+  //     method: 'DELETE',
+  //     url: '/api/venues/'+$(this).attr('data-id'),
+  //     success: deleteVenueSuccess,
+  //     // error: deleteBookError
+  //   });
+  // });
+
+});
+
+function getVenueHtml(venue) {
```

Comment:
Make sure you're indenting properly - even when you're writing a template literal string like this! It makes it a lot easier to debug and insures the code is correct when it's appended to your index.html.